### PR TITLE
Adds documentation for git rebase and git fetch

### PR DESCRIPTION
**Pull request by sethwoodworth**

### CODE REVIEW

**Comment by waldyrious** [comment]
Good start, git is one of the commands that benefits the most from TLDR :+1:

File: pages/common/git-rebase.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# git rebase
+
+> Apply local commits on top of another branch's history
+
+- Rebase your local branch with the latest changes in master
```
Comment (igorshubovych) [inline pages/common/git-rebase.md]: I would mention that it is an interactive rebase

File: pages/common/git-rebase.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+`git rebase -i master` (Then follow the directions in the terminal for merge conflicts)
```
Comment (igorshubovych) [inline pages/common/git-rebase.md]: The text in parens should not be here according to file format.

**Comment by sethwoodworth** [comment]
@igorshubovych style changed!

File: pages/common/git-rebase.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# git rebase
+
+> Apply local commits on top of another branch's history
+
+- Rebase your local branch interactively with the latest changes in local master
+
+`git rebase -i master`
+
+- Rebase your local branch  interactively with the latest changes from upstream
```
Comment (ostera) [inline pages/common/git-rebase.md]: Double space between `branch` and `interactively`

**Comment by ostera** [comment]
Very nice work @sethwoodworth! Fix that double space, squash your changes and you'll be all set :)

**Comment by rubenvereecken** [comment]
:tada:

**Pull request merged**